Services/WindowServer/WindowManager.cpp

```diff
@@ -163,6 +163,22 @@ Gfx::IntSize WindowManager::resolution() const
     return Screen::the().size();
 }
 
+void WindowManager::set_acceleration_factor(double factor)
+{
+    Screen::the().set_acceleration_factor(factor);
+    dbgln("Saving acceleration factor {} to config file at {}", factor, m_config->file_name());
+    m_config->write_entry("Mouse", "AccelFactor", String::formatted("{}", factor));
+    m_config->sync();
+}
+
+void WindowManager::set_scroll_scalar(unsigned scalar)
+{
+    Screen::the().set_scroll_scalar(scalar);
+    dbgln("Saving scroll scalar {} to config file at {}", scalar, m_config->file_name());
+    m_config->write_entry("Mouse", "ScrollScalar", String::number(scalar));
```

Comment:
What's a "scalar" in this context? Can we find a better word? ScrollWheelStep? ScrollStepSize?

Reply:
i think "ScrollStepSize" is fitting, should i change it in the IPC def as well, or just here?

Reply:
also done